src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/config/FeesConfiguration.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.iacaseapi.infrastructure.config;
+
+import java.util.HashMap;
+import java.util.Map;
+import lombok.*;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Getter
+@Setter
```

Comment:
Do we need a setter here?  Once the class is instantiated we probably don't want to allow anything to change it.

Reply:
spring probably just needs a constructor